pkg/apis/resource/types.go

```diff
@@ -988,6 +988,14 @@ type ResourceClaimTemplateList struct {
 	Items []ResourceClaimTemplate
 }
 
+const (
+	AllocatedDeviceStatusMaxConditions        int = 8
+	AllocatedDeviceStatusDataMaxLength        int = OpaqueParametersMaxLength
+	NetworkDeviceDataMaxIPs                   int = 16
+	NetworkDeviceDataInterfaceNameMaxLength   int = 256
+	NetworkDeviceDataHardwareAddressMaxLength int = 128
```

Comment:
Can you add comments that link these constants to the corresponding fields?

We cannot reference the constants in the field documentation because users looking at the public API docs won't see them, but here we can create that link.

Reply:
Thank you, I added the comments